cypress/e2e/cloud/buckets.test.ts

```diff
@@ -174,68 +241,40 @@ describe('Explicit Buckets', () => {
       })
   })
 
-  it('should be able to create an explicit bucket and add schema file during editing', function() {
-    cy.getByTestID('Create Bucket').click()
-    cy.getByTestID('bucket-form-name').type('explicit_bucket')
-    cy.getByTestID('accordion-header').click()
-    cy.getByTestID('explicit-bucket-schema-choice-ID').click()
+  it('should be able to create an explicit bucket and add json schema file during editing', function() {
+    const origFileContents = `[{"name":"time","type":"timestamp"},
+        {"name":"fsWrite","type":"field","dataType":"float"} ]`
 
-    cy.getByTestID('bucket-form-submit').click()
-
-    cy.getByTestID(`bucket-card explicit_bucket`)
-      .should('exist')
-      .within(() => {
-        cy.getByTestID('bucket-settings').click()
-      })
-    cy.getByTestID('accordion-header').click()
+    const checkContents = (cy: Cypress.Chainable) => {
+      cy.readFile(`cypress/downloads/first_schema_file.json`)
```

Comment:
where is this file defined? i can't find it anywhere in the codebase.

this test looks suspiciously tautological - it's just echoing back the object you give it. Is it reading in an actual json file?

Reply:
it actually isn't.  the csv is converted to an object and sent to the backend; 
then the backend converts it back into csv.  so it actually isn't.

Reply:
the file is created when the user presses the download button.  see the test that this method is using.

Reply:
i'm not sure what you want; if you would like me to do this let's chat/please give me a more detailed example.